Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -453,14 +472,12 @@ void MemoryWidget::OnSearchAddress()
   // Returns 0 if conversion fails
   const u32 addr = m_search_address->text().toUInt(&good_addr, 16);
   const u32 offset = m_search_offset->text().toUInt(&good_offset, 16);
-
   QFont addr_font, offset_font;
   QPalette addr_palette, offset_palette;
 
   if (good_addr || m_search_address->text().isEmpty())
   {
-    if (good_addr)
-      m_memory_view->SetAddress(addr + offset);
+    m_memory_view->SetAddress(addr + offset);
```

Comment:
This condition actually did something, if the textbox is empty this now sets address 0 instead of not doing anything.

Reply:
Ah sorry. I'll revert and slightly rewrite it to make it more clear.